apps/demo/src/Home.tsx

```diff
@@ -104,7 +109,13 @@ export const Home = ({ changePage }: PageProps) => {
               id: "isGuest",
               headerName: "Is Guest",
               renderCell: (params) => {
-                return params.row.isGuest ? "yes" : "no";
+                return params.row.isGuest ? (
```

Comment:
global: WDYT about showing a built-in confirmation modal on character delete click?

Reply:
It's a good idea, but for now I'm not able to import Confirmation Modal. Let's do that later : )

https://github.com/openfun/cunningham/issues/262